cpp/ql/lib/experimental/semmle/code/cpp/semantic/analysis/ModulusAnalysis.qll

```diff
@@ -146,7 +146,7 @@ module ModulusAnalysis<DeltaSig D, UtilSig<D> U> {
   private predicate phiSelfModulus(
     SemSsaPhiNode phi, SemSsaVariable inp, SemSsaReadPositionPhiInputEdge edge, int mod
   ) {
-    exists(SemSsaBound phibound, int v, int m |
+    exists(Bounds::SemSsaBound phibound, int v, int m |
```

Comment:
Again, it would be awesome if we could just `import Bounds` to avoid all these prefixes 🙏.